.gitpod.yml

```diff
@@ -3,9 +3,21 @@ additionalRepositories:
   - url: https://github.com/mattermost/mattermost-webapp
 
 ports:
+  # 8065 - for Mattermost instance
   - port: 8065
+    description: 8065 - for Mattermost instance
```

Comment:
To be consistent with the description below:

```suggestion
    description: for Mattermost instance
```

Also I think the comments can be removed here since they are redundant with the descriptions

Reply:
Where is this description shown? I don't see it in the list of ports:

<img width="347" alt="image" src="https://user-images.githubusercontent.com/6913320/208541582-2a5268aa-1ff6-4156-8c53-937d0a5fde9d.png">

Reply:
@mickmister it's supposed to show up in the Ports tab next to the Terminal tab (see [Specify port names & descriptions](https://www.gitpod.io/docs/configure/workspaces/ports#specify-port-names--descriptions))

Reply:
Ok I see that it is shown in this view as well if you drag it open enough (not that it would make sense to do that)
<img width="1788" alt="image" src="https://user-images.githubusercontent.com/6913320/208599771-e0d29e17-8b73-4770-82f8-4ff06527c5bd.png">

Also, you can apparently drag the `PORTS` view back and forth between the left sidebar and the tabs near the terminal